monasca/templates/_keystone_env.tpl

```diff
@@ -23,6 +23,17 @@ value: "{{ . }}"
 {{- end }}
 {{- end }}
 
+{{- define "monasca_mysql_secret_env" }}
```

Comment:
why is this in the keystone env file? Seems related to mysql almost makes sense to put into another file

Reply:
Ill rename the file so it makes more sense